pallets/file-system/src/lib.rs

```diff
@@ -199,6 +201,10 @@ pub mod pallet {
         #[pallet::constant]
         type MaxBspsPerStorageRequest: Get<u32>;
 
+        /// Maximum number of data servers that can be registered for a move bucket request.
+        #[pallet::constant]
+        type MaxDataServersPerMoveBucketRequest: Get<u32>;
```

Comment:
I removed this and instead created a double map from bucket to data servers. Deleting is simple since I can just delete the bucket id which is the prefix.